style.css

```diff
@@ -0,0 +1,298 @@
+@font-face {
+  font-family: 'Olde English';
+  src: url('Olde English.ttf');
+}
+
+header 
+{
+	position  : fixed;
+	padding   : 15px;
+	top       : 0px;
+	left      : 0px;
+	right     : 0px;
+	display   : flex;
+	height: 25px;
+	border-bottom: 1px solid #DDD;
+	background-color: white;
+}
+
+header *:hover {
+	cursor: pointer;
+}
+
+.NewYorkTitle
+{
+	font-family: 'Olde English';
+	font-size: 29px;
+	margin   :0;
+	margin-top: -5px;
+	margin-left: 20%;
+}
+header ul 
+{
+	margin: 0px;
+}
+header ul li 
+{
+   display: inline-block;
+   margin-right: 15px;
+
+}
+#logButton
+{
+	position : absolute;
+	top : 10px;
+	right: 20px;
+	margin-left : 12px;
+	margin-top : 10;
+}
+
+ button 
+{
+	padding : 10px;
+	background: #2980b9;
+	color : white;
+	font-weight: bold;
+	font-size: 10px;
+	border-radius: 3px;
+
+
+}
+#titleOfArticle {
+	margin-top: 100px;
+	display : block;
+	width : 53%;
+	margin-left: 100px;
+	font-family: georgia;
+}
+
+#container {
+	display: block;
+	margin: 100px auto;
+	width: 550px;
+}
+
+#image1 {
+	width: 800px;
+	display: block;
+	margin: 30px auto;
+}
+
+#image1 img {
+	height: 500px;
+	width: auto;
+}
+#biggerP {
+	font-size: 16px;
+	/*margin: 10px ;*/
+	width: 95%;
+	color: grey;
+}
+#smallerP {
+	font-size: 13px;
+	color: grey;
+	margin-top: -10px;
+}
+
+#smallGrey {
+	font-size: 15px;
+	color: grey;
+}
+
+	 h5 {
+	font-size: 17px;
+	color: grey;
+	font-weight: normal;
+	margin: 0px;
+}
+
+#video p {
```

Comment:
Wouldn't it be better to target the p elements you want to behave this way with a class?